command/drivetrain.py

```diff
@@ -105,7 +108,11 @@ def execute(self) -> None:
         self.zero()
 
     def isFinished(self) -> bool:
-        return self.zero_success()
+        # self.subsystem.gyro.reset_angle()
```

Comment:
Where did these changes come from? Are ya'll merging each others unfinished code?

Reply:
Sebastian is merged his elevator code.